src/components/memberships/MembershipsPage.js

```diff
@@ -10,34 +10,66 @@ class MembershipsPage extends React.Component {
   componentDidMount() {
     this.props.loadMemberships(this.props.user_email, this.props.f2f_date)
   }
+
+  copyEmails = () => {
+    this.input.select();
+    document.execCommand('copy');
+    this.input.blur();
+  }
+
+  cardHeight = (elements) => {
+    const levelsCount = Math.ceil(elements.length / 3);
+    const firstLevelHeight = 284;
+    const nextLevelHeight = 102;
+    if (levelsCount < 1 ) {
+      return firstLevelHeight;
+    }else{
+      return firstLevelHeight + (levelsCount-1) * nextLevelHeight
+    }
```

Comment:
There is a way to use method for automatically measuring items - but then animations stop working. For now I will leave in that way.